### PR TITLE
doc: add release notes for 1.7.0

### PR DESCRIPTION
**Pull request by greg-fer**
Added the release notes file and other required edits.
NCSDK-11024

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

----

- [x] Add entry for the cross-search
- [x] Production/development statement to be checked -- confirmed at the doc review meeting: no changes needed for 1.7.0
- [x] Documentation section to be updated
- [x] mcumgr section to be checked by @nvlsianpu and @de-nordic 
- [x] Entry for #5410 
- [x] Merge in entry for #5467 (also known issues)
- [x] Merge in entry for #5577 
- [x] Merge in entry for #5511 
- [x] Add mention of CMake update to 3.20
- [x] Update based on changelog History edits for Sept 10
- [x] Check #5479 and #5581 if in (not merged, no update made)
- [x] Add entry for #5547 from @zycz 
- [x] Add highlights from @martiles (mail thread)
- [x] Add #5637 entry from https://github.com/greg-fer/fw-nrfconnect-nrf-1/pull/7
- [x] Add entry for #5640 from @jciupis 
- [x] Add entry for #5648 
- [x] Add entry for #5654 
- [x] Update based on changelog History edits for Sept 20
- [x] One last check for commit IDs for MCUboot, mcumgr (done), and Zephyr

### CODE REVIEW

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-1.7.0.rst:15]: Hi, @jdstapes , do these lines need to be updated? Checking as per https://projecttools.nordicsemi.no/jira/browse/NCSDK-11021 -- "Check/update the development/production support statement". Thanks!

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-1.7.0.rst:437]: WIP

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-1.7.0.rst:391]: @nvlsianpu , @de-nordic , should we add any updates here?

File: doc/nrf/releases/release-notes-changelog.rst
Comment (melwee01) [inline doc/nrf/releases/release-notes-changelog.rst:20]:
```suggestion
nRF Connect for Cloud is now nRF Cloud.
```

File: doc/nrf/releases/release-notes-changelog.rst
Comment (melwee01) [inline doc/nrf/releases/release-notes-changelog.rst:110]:
```suggestion
* Updated:
  * nRF Connect for Cloud is now nRF Cloud.
```

File: doc/nrf/releases/release-notes-changelog.rst
Comment (grochu) [inline doc/nrf/releases/release-notes-changelog.rst:20]: Please move the Find My entry to the Highlights section in release-notes-1-7-0.rst. In fact in release-notes-changelog.rst there used to be no "Highlights" section. I added it as part of the 1.7.0 release notes PR to mention "Find My".

**Review by grochu — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-changelog.rst:20]: Please check doc/nrf/releases/release-notes-1.7.0.rst, not the changelog.

Comment (greg-fer) [reply doc/nrf/releases/release-notes-changelog.rst:20]: Highlights are populated by the SWGMs and PMTs, so I've only applied the change to the documentation section.

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:391]: Edits provided by @de-nordic , thanks!

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (melwee01) [inline doc/nrf/releases/release-notes-1.7.0.rst:57]:
```suggestion

* nRF Connect for Cloud is now nRF Cloud.

```

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (MarekPorwisz) [inline doc/nrf/releases/release-notes-1.7.0.rst:237]: Added support for nRF21540 supported ... reads kind of strange. can we somehow rephrase it?

**Review by trond-snekvik — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (mia-ko) [inline doc/nrf/releases/release-notes-1.7.0.rst:219]: Need to add migration details for mesh

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (mia-ko) [inline doc/nrf/releases/release-notes-1.7.0.rst:33]: Thingy:53 is not added for all these mesh samples
Checking with PMT if all should have.

**Review by mia-ko — CHANGES_REQUESTED** [review]

**Review by pdunaj — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:219]: Please add as a comment here or a PR to this PR.

Comment (mia-ko) [reply doc/nrf/releases/release-notes-1.7.0.rst:219]: Waiting for this PR to get merged first: https://github.com/nrfconnect/sdk-nrf/pull/5156

Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:219]: Migration now added.

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (LuDuda) [inline doc/nrf/releases/release-notes-1.7.0.rst:39]: Thread examples are not supported for Thingy.

**Review by LuDuda — APPROVED** [review]
:+1: for Matter part

**Review by grochu — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (thomasstenersen) [inline doc/nrf/releases/release-notes-1.7.0.rst:51]: This should specify for which protocol stack. I.e. 802.15.4 (@czeslawmakarski).

File: doc/nrf/releases/release-notes-changelog.rst
Comment (thomasstenersen) [inline doc/nrf/releases/release-notes-changelog.rst:41]: Fix the merge conflict :)

**Review by thomasstenersen — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-changelog.rst:41]: How did this get here... Thanks!

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:439]: Updated.

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:33]: Removed three out of five samples after receiving an update from @Balaklaka

**Review by mia-ko — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (mia-ko) [inline doc/nrf/releases/release-notes-1.7.0.rst:447]: It seems like this line is causing the doc build to fail...

**Review by Balaklaka — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:15]: Checked at the doc review meeting. No changes needed.

**Review by thomasstenersen — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-1.7.0.rst:462]:
```suggestion
  * Added new documentation sets for `Trusted Firmare-M <TF-M documentation_>`_ and :doc:`Matter <matter:index>`.
```

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-1.7.0.rst:515]:
```suggestion
      * :ref:`thread_ug_supported_features` - Updated with new supported features.
```

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-1.7.0.rst:589]:
```suggestion
  * Renamed the "AT commands" page to :ref:`at_socket`
  * :ref:`nrfxlib:nrf_modem_at` - Page added.
```

File: doc/nrf/ug_tfm.rst
Comment (umapraseeda) [inline doc/nrf/ug_tfm.rst:24]:
```suggestion
For official documentation, see the `TF-M documentation`_ set.
```

**Comment by FrancescoSer** [comment]
PR #5479 and #5581 might need their changelog entry moved here, if merged before this PR goes in.

**Comment by greg-fer** [comment]
> PR #5479 and #5581 might need their changelog entry moved here, if merged before this PR goes in.

No worries, I check the file history for changelog daily.

**Comment by maciejpietras** [comment]
Can you please add to Zigbee: 
"Added development support for :ref:`radio front-end module (FEM) <ug_radio_fem>` for nRF53 Series devices and nRF21540 EK."

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-1.7.0.rst:462]: Link not working
```suggestion
  * Added new documentation sets for `Trusted Firmare-M <TF-M documentation_>`_ and :doc:`Matter <matter:index>`.
```

**Review by hakonfam — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Review by MarekPorwisz — APPROVED** [review]

**Review by oyvindronningstad — APPROVED** [review]

**Review by nordic-hani — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (krlo) [inline doc/nrf/releases/release-notes-1.7.0.rst:37]: Change to: ":ref:`ug_radio_fem` for the nRF21540 FEM range extender is now supported for production."

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (krlo) [inline doc/nrf/releases/release-notes-1.7.0.rst:40]: Change to: "Support for Bluetooth® LE, Thread, Zigbee and multiprotocol applications and samples using :ref:`nrfxlib:mpsl`."

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (krlo) [inline doc/nrf/releases/release-notes-1.7.0.rst:47]: Add this bullet: "Added support for nRF Connect SDK development in nRF Connect for Visual Studio Code"

Comment (krlo) [inline doc/nrf/releases/release-notes-1.7.0.rst:47]: Add this bullet: "Added production support for nRF52833 for Bluetooth LE HomeKit accessories."

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (krlo) [inline doc/nrf/releases/release-notes-1.7.0.rst:262]: Add this information: "In this release, Zigbee is supported for development and should not be used for production. nRF Connect SDK v1.5.1 contains the certified Zigbee solution supported for production."

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (krlo) [inline doc/nrf/releases/release-notes-1.7.0.rst:280]: Change to: ":ref:`ug_zigbee_tools_ncp_host` is now supported for production and moved to version 1.0.0."

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (krlo) [inline doc/nrf/releases/release-notes-1.7.0.rst:302]: Change to: "Wi-Fi coexistence functionality supported for development."

**Review by krlo — CHANGES_REQUESTED** [review]

**Review by jdstapes — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (jciupis) [inline doc/nrf/releases/release-notes-1.7.0.rst:310]:
```suggestion
  * Updated ``NRF_802154_SWI_PRIORITY``, ``NRF_802154_ECB_PRIORITY`` and ``NRF_802154_SL_RTC_IRQ_PRIORITY`` to comply with the limitation introduced in :ref:`nrfxlib:nrf_802154_driver`.
```

Not sure if the syntax is correct, but this entry is related with https://github.com/nrfconnect/sdk-nrf/pull/5640.

**Review by shantha-14 — CHANGES_REQUESTED** [review]
NEeds some decision with thingy53 highlights

**Review by jciupis — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-1.7.0.rst:414]: @de-nordic , please confirm this is the correct commit ID.

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-1.7.0.rst:401]: @sigvartmh , @hakonfam , can you confirm this is the correct commit ID?

**Review by jhn-nordic — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-1.7.0.rst:425]: @mbolivar-nordic , can you confirm this is the correct commit ID? One last check before merge.

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:402]: Updated ``657deb65`` after checking on Teams.

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (greg-fer) [reply doc/nrf/releases/release-notes-1.7.0.rst:335]: Added, please check.

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (shantha-14) [reply doc/nrf/releases/release-notes-1.7.0.rst:403]: @SebastianBoe to confirm

**Review by Vge0rge — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Review by krlo — APPROVED** [review]

File: doc/nrf/releases/release-notes-1.7.0.rst
Comment (SebastianBoe) [reply doc/nrf/releases/release-notes-1.7.0.rst]: Was the wrong commit. Fixed now.

**Review by SebastianBoe — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Pull request merged**
